nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+    "verbose": false,
+    "ignore": [
+        "*.test.*",
+        "dist/*",
+        "node_modules/*"
+    ],
+    "ext": "js,ts,ejs,json,yml"
+}
```

Comment:
```suggestion
}

```

Reply:
*at myself* You see not \n terminating a file one time cause a rather spectacular bug and you never forget it.